modules/api/API.js

```diff
@@ -0,0 +1,335 @@
+import { PermissionHelper } from '../helper/PermissionHelper.js';
+import { LootSheetNPC5eHelper } from "../helper/LootSheetNPC5eHelper.js";
+import { MODULE } from '../data/moduleConstants.js';
+import { LootPopulator } from '../classes/LootPopulator.js';
+import { TableRoller } from '../classes/tableRoller.js';
+import { LootProcessor } from '../classes/LootProcessor.js';
+import { CurrencyHelper } from '../helper/currencyHelper.js';
+
+/**
+ * @description The lootsheet API
+ *
+ * @module lootsheetnpc5e.API
+ *
+ * @title Lootsheet NPC 5e API
+ * @version 1.0.0
+ */
+class API {
```

Comment:
- Classes composed entirely of static methods tend to be an antipattern - it usually means that all of the static methods should just be module-level functions (a module itself is quite similar to a class anyway). A module identifier like `API` can still be used via `import * as API from "../api/API.js"`.
- Since the only file in the "api" is "API.js", perhaps it would be better to remove the directory for now? Given the module's name, the additional directory is somewhat redundant.

Reply:
Thanks for the feedback.
In that case I'd prefer a version where the API calls are split to dedicated parts.

I will think about how and in which way to change that.
If you have examples of other foundry vtt modules that offer examples and
have a reasonable "abstractable" structure please feel free to link or comment.